script/xmlconverter.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import os
+import pathlib
+import sys
+import parmap
+from lxml.etree import XMLSyntaxError
+from xml2xml import xml2xml
+
+"""
+Script to run the xmlc onversion tools
+
+ License:
+     This file is part of the S-ENDA-Prototype repository (https://github.com/metno/py-mmd-tools).
```

Comment:
please change to `License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)`